pages/phases/certificate.md

```diff
@@ -20,16 +20,31 @@ roles: [hr, hiring-mgr]
   {% include toolkit/certificate.html %}
 </div>
 
-## HR Preference Adjudication
+`<TO BE MOVED TO 'HOW OUR PROCESS IS DIFFERENT'>`
```

Comment:
@humancompanion-usds - This needs to go before the "table" in "How Our Process is Different". I can't quite figure out what that HTML should look like, can you add some commits here to do that? 🙇‍♀